### DISCUSSION TITLE
调整 vmstorage 磁盘 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1066] [Announcements]
# 调整 vmstorage 磁盘 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/insight/best-practice/modify-vmstorage-disk/

**Comment by nicksors** [comment]
开启存储池扩容截图重复了吧

**Comment by windsonsea** [comment]
嗯嗯，刚提了 PR #1065
